chef/cookbooks/nova/templates/default/nova.conf.erb

```diff
@@ -3812,6 +3817,136 @@ cluster_name="<%= cluster %>"
 #integration_bridge=br-int
 
 
+[zvm]
```

Comment:
IIRC, the sections in the config files are alphabetically sorted. Could you keep it that way? :-)